radicle-httpd/src/api/v1/stats.rs

```diff
@@ -0,0 +1,24 @@
+use axum::response::IntoResponse;
+use axum::routing::get;
+use axum::{Extension, Json, Router};
+use serde_json::json;
+
+use crate::api::error::Error;
+use crate::api::Context;
+
+pub fn router(ctx: Context) -> Router {
+    Router::new()
+        .route("/stats", get(stats_handler))
+        .layer(Extension(ctx))
+}
+
+/// Return the stats for the node.
+/// `GET /stats`
+async fn stats_handler(Extension(ctx): Extension<Context>) -> impl IntoResponse {
+    let storage = &ctx.profile.storage;
+    let projects = storage.projects()?.len();
+
+    Ok::<_, Error>(Json(
+        json!({ "projects": { "count": projects }, "users": { "count": 0 } }),
```

Comment:
`"users": { "count": 0 }` atm. let's come up with a solution for this, or remove it entirely.